src/Microsoft.AspNetCore.Mvc.Core/FileContentResult.cs

```diff
@@ -54,6 +52,43 @@ public FileContentResult(byte[] fileContents, MediaTypeHeaderValue contentType)
         }
 
         /// <summary>
+        /// Creates a new <see cref="FileContentResult"/> instance with
+        /// the provided <paramref name="fileContents"/> and the
+        /// provided <paramref name="contentType"/>.
+        /// </summary>
+        /// <param name="fileContents">The bytes that represent the file contents.</param>
+        /// <param name="contentType">The Content-Type header of the response.</param>
+        /// <param name="lastModified">The <see cref="DateTimeOffset"/> of when the <see cref="FileContentResult"/>
```

Comment:
nit: `of when the <see cref="FileContentResult"/>` this should reference the parameter 'fileContents'. You can use `paramref`